about.html

```diff
@@ -1,5 +1,26 @@
 --- 
 layout: default
 title: About a11y Berlin
-description: a11y Berlin is an usergroup about accessibility and inclusive design.
+description: a11y Berlin is a usergroup about accessibility and inclusive design.
 ---
+
+<p>
+  A11y Meetup Berlin is designed to bring together the accessibility and inclusive design community to share and discuss ideas, best practices, and experiences. 
+  We would love to invite everyone interested in these topics.
+</p>
+
+<aside>
+  <h2>What does "a11y" mean?</h2>
```

Comment:
It's really nice that we have the explanation of the acronym, but maybe we should also have a short explanation on what web accessibility means e.g.

Reply:
@radibit Opened an issue for this.